src/site/content/en/identity/passkey-form-autofill/index.md

```diff
@@ -23,163 +23,264 @@ create a unified experience.
+passkeys, so asking users to select which method to use up front would be poor
+UX.
 
-Passkeys are also a new technology. Explaining them and making sure users are
-comfortable using them can be a challenge for websites. We can rely on familiar
+Passkeys are also a new technology. Explaining them and making sure users are 
+comfortable using them can be a challenge for websites. We can rely on familiar 
 user experiences for autofilling passwords to solve both problems.
 
 ## Conditional UI
 
-To create an efficient user experience for both passkey and password users, you
-can optimize the authentication flow using the browser's form autofill
-functionality. This is called
-[conditional UI](https://github.com/w3c/webauthn/wiki/Explainer:-WebAuthn-Conditional-UI)
-and it's a part of [the WebAuthn standard](https://w3c.github.io/webauthn/). A
-website can include passkeys in autofill suggestions.
-
-1.  The user lands on the sign-in page.
-1.  The user selects the username input field.
-1.  An account selector dialog pops up.
-1.  The user selects an account, whether that account uses a password or a
-    passkey.
-1.  The user verifies with fingerprint, face or device PIN.
-1.  The user is signed in.
+To build an efficient user experience for both passkey and password users, you
+can include passkeys in autofill suggestions. This is called [conditional
+UI](https://github.com/w3c/webauthn/wiki/Explainer:-WebAuthn-Conditional-UI) and
+it's a part of [the WebAuthn standard](https://w3c.github.io/webauthn/).
+
+<figure>
+  {%
+    Video src="video/YLflGBAPWecgtKJLqCJHSzHqe2J2/ZaxgMtkOh5CX2B1ZT9gd.mp4",
+    autoplay="true", loop="true"
+  %}
+</figure>
 
 As soon as the user taps on the username input field, an autofill suggestion
 dialog pops up which highlights the stored passkeys along with password autofill
 suggestions. The user can then choose an account and use the device screen lock
 to sign in.
 
-This way, users can sign in to your website with the existing form as if
-nothing has changed, but with
-[the added security benefit of passkeys](https://developers.google.com/identity/passkeys#security-considerations)
+This way, users can sign in to your website with the existing form as if nothing
+has changed, but with [the added security benefit of
+passkeys](https://developers.google.com/identity/passkeys#security-considerations)
 if they have one.
 
-{% Aside %}
+{% Aside 'success' %}
 
-When a user signs in with a password, consider using that opportunity to
-offer them to
-[create a passkey](/passkey-registration).
+When a user signs in with a password, consider using that opportunity to offer
+them to [create a passkey](/passkey-registration).
 
 {% endAside %}
 
+## How it works
+
+To authenticate with a passkey, you use [the WebAuthn 
+API](https://w3c.github.io/webauthn/).
+
+In a passkey authentication flow, four participants are involved in addition to 
```

Comment:
Would it make more sense to describe this as "The four components in a passkey authentication flow are:"?

Do we have to mention the user here?

Reply:
Four components sounds good to me. I don't have strong opinion on if adding a user is a good idea or not, but I omitted it because it was obvious.

```suggestion
The four components in a passkey authentication flow are:
```

Feel free to modify the suggesion and resolve.